front-end/src/reducers/i18n.js

```diff
@@ -0,0 +1,44 @@
+const translations = {
```

Comment:
consider having structure as fields, and translation as values
For example:
```javascript
locales: {
'en': 'en'
...
}, {
navigationLabel: {'en': 'Page1', 'fi': 'Sivu #1'}
}
```

This is just a suggestion. Use it or not depends on you.